data/campaigns/Eastern_Invasion/scenarios/01_Eastern_Invasion.cfg

```diff
@@ -0,0 +1,948 @@
+            [/then]
+        [/if]
+        [store_unit] # hide any unit on 10,15 so it's easier to see where the trapdoor is
+            [filter]
+                x,y=10,15
+            [/filter]
+
+            variable=trapdoor_unit
+            kill=yes
+        [/store_unit]
+
+        {HIGHLIGHT_IMAGE 10 15 scenery/trapdoor-open-highlight.png scenery/trapdoor-open.png}
+
+        [if] {VARIABLE_CONDITIONAL trapdoor_unit.length greater_than 0}
+            [then]
+                [unstore_unit]
+                    variable=trapdoor_unit
+                    animate=no
+                [/unstore_unit]
+                {CLEAR_VARIABLE trapdoor_unit}
+            [/then]
+        [/if]
+
+        [message]
+            speaker=Gweddry
+            message= _ "But we have orders to hold this outpost!"
+        [/message]
+        [message]
+            speaker=Dacyn
+            message= _ "You have not yet seen even one-hundredth of the undead armies. This outpost is already lost. Your death - as heroic as I am sure it would be - will benefit no one. Now come, there is little time to spare."
+        [/message]
+        [modify_turns]
+            value=-1
+        [/modify_turns]
+        {VARIABLE trapdoor_open yes}
+        [show_objectives]
+        [/show_objectives]
+
+#define VICTORY_CHOICE
+    [message]
+        side,x,y=1,10,15
+        message= _ "I’m at the trapdoor! Anyone still alive, follow me!"
+        [option]
+            label= _ "Escape"
+            [command]
+                [hide_unit]
+                    id=$unit.id
+                [/hide_unit]
+                [redraw][/redraw]
+                [endlevel]
+                    result=victory
+                    bonus=no
+                    {NEW_GOLD_CARRYOVER 40}
+                [/endlevel]
+            [/command]
+        [/option]
+        [option]
+            label= _ "Wait"
+        [/option]
+    [/message]
+#enddef
+        # in case you already have a unit on the trapdoor
+        [if]
+            [have_unit]
+                side,x,y=1,10,15
+            [/have_unit]
+
+            [then]
+                {VICTORY_CHOICE}
+            [/then]
+        [/if]
+
+        # you win when you move to the trapdoor
+        [event]
+            name=moveto
+            first_time_only=no
+
+            [filter]
+                side,x,y=1,10,15
+            [/filter]
+
+            {VICTORY_CHOICE}
+        [/event]
+    [/event]
+
+    # You are urged to hurry
+    [event]
+        name=turn 14
+        [message]
+            speaker=Dacyn
+            message= _ "If you wish to die, there will be plenty of time for that later."
+        [/message]
+    [/event]
+
+    #------------------
+    # LAST BREATH EVENTS
+    #------------------
+    [event]
+        name=last breath
+
+        [filter]
+            id=Nakeg-alvan
+        [/filter]
+
+        [message]
+            speaker=Nakeg-alvan
+            message= _ "..."
```

Comment:
I think the existing Wesnoth convention is to use three full stops rather than the actual ellipsis character.

https://wiki.wesnoth.org/Typography_Style_Guide#Ellipses